<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>671</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="loadButton">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>360</y>
      <width>141</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Database</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addToCart">
    <property name="geometry">
     <rect>
      <x>268</x>
      <y>360</y>
      <width>141</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Add to Cart</string>
    </property>
   </widget>
   <widget class="QPushButton" name="showCart">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>360</y>
      <width>161</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Cart</string>
    </property>
   </widget>
   <widget class="PetDatabase" name="petTable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>611</width>
      <height>211</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>150</number>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>41</number>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>41</number>
    </attribute>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Class</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Price</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Special Attribute</string>
     </property>
    </column>
   </widget>
   <widget class="BundleDatabase" name="bundleTable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>611</width>
      <height>111</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>200</number>
    </attribute>
    <column>
     <property name="text">
      <string>Bundle</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Price</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Savings</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>671</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PetDatabase</class>
   <extends>QTableWidget</extends>
   <header>petdatabase.h</header>
  </customwidget>
  <customwidget>
   <class>BundleDatabase</class>
   <extends>QTableWidget</extends>
   <header>bundledatabase.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>petTable</tabstop>
  <tabstop>bundleTable</tabstop>
  <tabstop>showCart</tabstop>
  <tabstop>loadButton</tabstop>
  <tabstop>addToCart</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
